_mot_conference/2015-06-18.md

```diff
@@ -0,0 +1,67 @@
+---
+speaker: Frédéric Merizen
```

Comment:
The speaker is not rendered within the page, maybe add him (a 👩 this year? 🤷‍♂️) as a signature?